src/components/Button/Button.tsx

```diff
@@ -18,46 +25,44 @@ export type ButtonProps = ButtonHTMLElementProps & {
    * Toggles clickable that fills the full width of its container
    */
   fullWidth?: boolean;
-  'data-testid'?: string;
-  size?: ClickableStyleProps<'button'>['size'];
+  /**
+   * The size of the button on screen
+   */
+  size?: Extract<Size, 'sm' | 'md' | 'lg'>;
   /**
    * Disables the field and prevents editing the contents
    */
   disabled?: boolean;
   /**
    * Loading state passed down from higher level used to trigger loader and text change
-   * @deprecated - This will be removed in a future release
```

Comment:
no longer deprecated?

Reply:
Yah a few things might need to be re-discussed in the new year. Right now, we don't have an alternative way to mark a button as "disabled b/c the app/page is doing something" so not very nice to remove without that :)